package.json

```diff
@@ -8,6 +8,7 @@
   ],
   "scripts": {
     "tsc": "tsc",
+    "test": "mocha",
```

Comment:
On some level it may be nice to keep all of our JS tests using Jest, but no problem if you're most familiar with mocha. Not a blocker in any way, just a thought I had :)

Reply:
Hm, setting up jest seems really nontrivial (the getting started guide... doesn't seem to work and throws a SyntaxError on its own config file?), and the intent is for this whole project to just go away in the future, so I'll stick with mocha for now